docs/json.md

```diff
@@ -176,10 +174,10 @@ This leads to the following generated JSON schema:
   "description": "Interface properties",
   "type": "object",
   "properties": {
-    "$schema": {
-      "title": "Schema",
+    "formatVersion": {
```

Comment:
The field schema has to be removed from the ABAP type `ty_main` which is defined above